core/src/test/java/org/apache/calcite/test/FoodmartTest.java

```diff
@@ -95,6 +100,22 @@
   // 2542: timeout. Running big, simple SQL cartesian product.
   //
 
+  @BeforeAll
+  public static void setupAsserts() {
+    assertFoodmart = CalciteAssert.that()
+      .with(CalciteAssert.Config.FOODMART_CLONE)
+      .pooled();
+    assertFoodmartLattice = CalciteAssert.that()
+      .with(CalciteAssert.Config.JDBC_FOODMART_WITH_LATTICE)
+      .pooled();
+  }
+
+  @AfterAll
+  public static void tearDownAsserts() {
+    assertFoodmart = null;
+    assertFoodmartLattice = null;
```

Comment:
Technically speaking it should be like "close the connection pool", but I don't think it matters much.